<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversions</class>
 <widget class="QMainWindow" name="Conversions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konwersja pliku do SqlLite</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Filtry</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="regex">
             <property name="text">
              <string>Regex dla tekstu:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="regexRecipe">
             <property name="text">
              <string>(?&lt;key&gt;[\\w ,]+):(?&lt;value&gt;[\\w ,]*);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="xquery">
             <property name="text">
              <string>Recepta XQuery dla xml</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="xqueryRecipe">
             <property name="plainText">
              <string>let $var1 := doc($inputDocument)/div/entry

let $var2 := ()

for $i in (1 to count($var1))
    let $var2 := concat ($var2, concat (string-join($var1[$i]/form/orth/&lt;p1&gt;{string()}&lt;/p1&gt;,','),':  ') , string-join($var1[$i]/sense/sense/sense/cit/quote/&lt;p2&gt;{string()}&lt;/p2&gt;,','), ';')

return $var2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Wykonaj powyższe czynności</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QPlainTextEdit" name="inFileText"/>
              <widget class="QTableWidget" name="afterFiltering">
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>170</number>
               </attribute>
               <column>
                <property name="text">
                 <string>Klucz</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Wartość</string>
                </property>
               </column>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Some output stuff</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="output"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtw_rz"/>
    <addaction name="actionZapisz_jako_sql"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionInstrukcja"/>
    <addaction name="separator"/>
    <addaction name="actionO_mnie"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOtw_rz">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionZapisz_jako_sql">
   <property name="text">
    <string>Zapisz jako sql</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionInstrukcja">
   <property name="text">
    <string>Instrukcja</string>
   </property>
  </action>
  <action name="actionO_mnie">
   <property name="text">
    <string>O mnie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
